soc/arm/actions/atb110x/soc_irq.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Actions (Zhuhai) Technology Co., Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file IRQ configuration macros for ATB110X
+ */
+
+#ifndef _ACTIONS_SOC_IRQ_H_
+#define _ACTIONS_SOC_IRQ_H_
+
+#define IRQ_ID_TIMER0         0
```

Comment:
I'd prefer we remove this file and get values from DT.

Reply:
Done